pallets/omnipool-liquidity-mining/src/lib.rs

```diff
@@ -369,7 +364,8 @@ pub mod pallet {
 				owner.clone(),
 				yield_per_period,
 				min_deposit,
-				lrna_price_adjustment,
+				//NOTE: oracle's price is used for `price_adjustment`
+				FixedU128::one(),
```

Comment:
Warehouse LM require `price_adjustment` as a param for case when oracle is not used for price adjustment and it's updated manually as on the Basilisk. 
One - value doesn't matter that much in this case as it's not used but it can't be zero because there is validation for it in the warehouse

Reply:
hm.. this looks like a bad workaround.  would not be better to have it as option or alternatively, having another interface where this does not need to specify  ?

i feel like this could be very misleading.